docs/source/getting-started/first-feature.rst

```diff
@@ -175,24 +179,22 @@ You can see we simply follow the formatting of the previous panels and include t
 That is all for the implementation!
 Let's give the new functionality a spin.
 
-Running the New App
-===================
+Running the New Feature
+=======================
 
-To start up the new app, open up your terminal application.
-First make sure your previously run commands are terminated, in particular the ``daml-start.sh`` command.
-You can do this by hitting ``Ctrl-C`` in the terminal window where you ran the command.
-This shuts down the previous instances of the sandbox and JSON API server: it is important that we start our new app with new instances of these components.
+We need to terminate the previous ``./daml-start.ch`` process and run it again, as we need to have a Sandbox instance with a DAR file containing the new feature. As a reminder, by running the ``./daml-start.ch`` again we will 
```

Comment:
```suggestion
We need to terminate the previous ``./daml-start.sh`` process and run it again, as we need to have a Sandbox instance with a DAR file containing the new feature. As a reminder, by running ``./daml-start.sh`` again we will 
```
Are you sure you are not Swiss :joy:

Reply:
😂I went back and forth between .sh and .ch so many times while writing it